.github/workflows/production.yml

```diff
@@ -61,6 +61,8 @@ jobs:
           K8S_SECRET_MAILER_EMAIL_BACKEND: "anymail.backends.mailgun.EmailBackend"
           K8S_SECRET_DEFAULT_FROM_EMAIL: "Kulttuurin kummilapset <noreply@hel.fi>"
           K8S_SECRET_ILMOITIN_TRANSLATED_FROM_EMAIL: "fi=Kulttuurin kummilapset <noreply@hel.fi>,en=Culture Kids <noreply@hel.fi>,sv=Kulturens fadderbarn <noreply@hel.fi>"
+          K8S_SECRET_NOTIFICATION_SERVICE_API_TOKEN: ${{ secrets.GH_NS_API_TOKEN_STABLE }}
```

Comment:
Should we add `TRANSLATED_SMS_SENDER` config here and staging?

Reply:
Maybe if we get the translations. :) Checked from the Palvelutarjotin API and it's not configured there either.

Currently it works like this: 
```
    if language in getattr(settings, "TRANSLATED_SMS_SENDER", {}):
        sender = settings.TRANSLATED_SMS_SENDER[language]
    else:
        sender = settings.DEFAULT_SMS_SENDER
```
Maybe the else should be replaced with `if not sender`?

Reply:
I will use the translations from the email
```
          K8S_SECRET_ILMOITIN_TRANSLATED_FROM_EMAIL: "fi=Kulttuurin kummilapset <noreply@hel.fi>,en=Culture Kids <noreply@hel.fi>,sv=Kulturens fadderbarn <noreply@hel.fi>"
```